.github/workflows/accessibility-alt-text-bot.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   accessibility_alt_text_bot:
     name: Check alt text is set on issue or pull requests
     runs-on: ubuntu-latest
```

Comment:
If adding permissions to the workflow in primer/react ([#3326](https://github.com/primer/react/pull/3326)) is successful I will add them here.